src/Services/publicApiService.js

```diff
@@ -22,4 +22,14 @@ const Get = () => {
     .catch((err) => console.log(err));
 };
 
+const Post = (route,body) => {
+  try{
+    const data = axios.post(`${route}`,body);
+    return data
+  }catch(error) {
+    console.log(error)
+  }
+}
+
+export {Post};
 export default Get;
```

Comment:
if you add the export {Post} please delete export default Get and add the into the export post, it should be something like;
```
export { Post, Get }
```
```

Reply:
Solved